.github/workflows/periodic.yml

```
@@ -26,3 +26,38 @@ jobs:
           title: Broken link detected by CI
           content-filepath: .github/ISSUE_TEMPLATE/broken-link.md
           labels: automated, broken link
+
+  test-bundle-periodic:
+    name: Operator Bundle Test
+    if: github.repository_owner == 'submariner-io'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v2
+
+      - name: Fetch all git tags
+        run: git fetch origin +refs/tags/*:refs/tags/*
+
+      - name: Generate Submariner version
+        run: |
+          VERSION=$(git describe --abbrev=0 --tags --match=v[0-9]*\.[0-9]*\.[0-9]* | cut -d'-' -f1 | cut -c2-)
+          FROM_VERSION=$(git describe --abbrev=0 --tags --match=v[0-9]*\.[0-9]*\.[0-9]* --exclude=?${VERSION}* | cut -d'-' -f1 | cut -c2-)
+          echo "VERSION=${VERSION}" >> $GITHUB_ENV
+          echo "FROM_VERSION=${FROM_VERSION}" >> $GITHUB_ENV
+          echo "CHANNEL=alpha-$(echo ${VERSION} | cut -d'.' -f1,2)" >> $GITHUB_ENV
+          echo "REPO=localhost:5000" >> $GITHUB_ENV
+          echo "IMG=${REPO}/submariner-operator:local" >> $GITHUB_ENV
+
+      - name: Run E2E with using the bundle deployment method
```

```suggestion
      - name: Run E2E using the bundle deployment method
```

Sorry my suggestion needed a suggestion :sweat_smile: